# HG changeset patch
# User Brian Gough <bjg@gnu.org>
# Date 1231824320 18000
#      Tue Jan 13 00:25:20 2009 -0500
# Node ID 5aafd61e4c3c7a771bbae8a542b13f79dda64d96
# Parent  433e788a015b143a0dd502974cdfa7d701056f56
[docs] @var{XI}, @var{YI}. => @var{xi}, @var{yi}.

diff --git a/scripts/general/interp2.m b/scripts/general/interp2.m
--- a/scripts/general/interp2.m
+++ b/scripts/general/interp2.m
@@ -27,17 +27,17 @@
 ## surface function. If @var{x} and @var{y} are vectors their length
 ## must correspondent to the size of @var{z}. @var{x} and @var{y} must be
 ## monotonic. If they are matrices they  must have the @code{meshgrid} 
 ## format. 
 ##
 ## @table @code
 ## @item interp2 (@var{x}, @var{y}, @var{Z}, @var{xi}, @var{yi}, @dots{}) 
 ## Returns a matrix corresponding to the points described by the
-## matrices  @var{XI}, @var{YI}. 
+## matrices @var{xi}, @var{yi}.  
 ##
 ## If the last argument is a string, the interpolation method can
 ## be specified. The method can be 'linear', 'nearest' or 'cubic'.
 ## If it is omitted 'linear' interpolation  is assumed.
 ##
 ## @item  interp2 (@var{z}, @var{xi}, @var{yi})
 ## Assumes @code{@var{x} = 1:rows (@var{z})} and @code{@var{y} = 
 ## 1:columns (@var{z})}
